airflow/example_dags/example_bash_operator.py

```diff
@@ -73,5 +73,10 @@
 # [END howto_operator_bash_skip]
 this_will_skip >> run_this_last
 
+from tests.system.utils import get_test_run  # noqa: E402
```

Comment:
Actually those lines are still needed I think. They are also using "dag.test" under the hood.

Reply:
(maybe we can rename them, but they are still neded to use pytest).

Reply:
i'm confused -- you say the lines are still needed but i am not removing anything here 😕 ?

Reply:
OH GOD .... 

Yeah.. Somewhat in my head those additions were .... removals :)

Friday night after returning back from holidays and having cold whole week 🤯

Reply:
lol know the feeling